src/main/java/com/happeningnow/model/Event.java

```diff
@@ -0,0 +1,43 @@
+package com.happeningnow.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.UuidGenerator;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+
+@Entity
+@Table(name = "event")
+public class Event {
```

Comment:
Looking to the proposed model, itis missing a reference for the organizer in this entity

Reply:
ok